data_scripts/get_all.sh

```diff
@@ -3,7 +3,8 @@
 # Run all migration scripts in appropriate order
 ./get_sites.py
 ./get_schools.py
-./get_users.py
+#./get_users.py
+./get_test_user.py
```

Comment:
Instead of copying and pasting code why don't you just update the ``get_user.py`` script to look for the 
 existence of ``test_users.csv`` and use that if that's there, otherwise continue the logic. We already do that for the csv's we have included in ansible vs. the repo.